test/e2e/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "private": true,
+  "name": "web-speed-test-wdio",
+  "version": "0.0.1",
+  "description": "Web speed test using Mocha, WebDriverIO, Allure report ",
+  "author": "Felix Zilber",
+  "license": "MIT",
+  "engines": {
+    "node": ">7.5.0"
```

Comment:
Does not match your `travis.yml` settings

Reply:
updated